py/client/pydeephaven/table.py

```diff
@@ -219,6 +220,23 @@ def sort(self, order_by: Union[str, List[str]], order: Union[SortDirection, List
         """
         return super().sort(order_by, order)
 
+    def sort_descending(self, order_by: Union[str, List[str]]) -> Table:
+        """The sort_descending method creates a new table where rows in a table are sorted in a largest to smallest
+        order based on the order_by column(s).
+
+        Args:
+            order_by (Union[str, List[str]]): the column name(s)
+
+        Returns:
+            a Table object
+
+        Raises:
+            DHError
+        """
+        order_by = to_list(order_by)
+        order = [SortDirection.DESCENDING] * len(order_by)
```

Comment:
```suggestion
        order = SortDirection.DESCENDING
```
I think this should be sufficient and is more simple.